theano/tensor/nnet/tests/test_sigm.py

```diff
@@ -339,7 +339,7 @@ def test_local_ultra_fast_sigmoid(self):
         topo = f.maker.fgraph.toposort()
         assert topo[0].op == ultra_fast_sigmoid
         assert len(topo) == 1
-        ux_v = f([[-50, -10, -4, -1, 0, 1, 4, 10, 50]])
+        # ux_v = f([[-50, -10, -4, -1, 0, 1, 4, 10, 50]])
```

Comment:
uncomment that. We want the call to be done. Just don't affect it to a variable.